app/schema/answer.py

```diff
@@ -100,3 +100,14 @@ def mandatory_error(self, state):
         state.errors = []
         state.errors.append(self.questionnaire.get_error_message('MANDATORY', self.id))
         state.is_valid = False
+
+    @staticmethod
+    def _valid_option_selected(state_input, options):
```

Comment:
This can be localised to `app/schema/answers/checkbox_answer.py` as it's not used anywhere else